TestHost/Program.cs

```diff
@@ -135,12 +128,16 @@ private static void DiscoverUnitTests(IEnumerable<ITestDiscoverer> tds, string s
                 timeFilter,
                 (string assemblyPath) =>
                 {
-                    var asmPath = FilePath.NewFilePath(assemblyPath);
                     var disc = new XUnitTestDiscoverer();
                     disc.TestDiscovered.AddHandler(
                         new FSharpHandler<TestCase>(
                             (o, ea) =>
                             {
+                                if (ignoredTests.Any(x => ea.FullyQualifiedName.StartsWith(x)))

```

Comment:
This super simple super cool. Can you please pass the ignoredTests to XUnitTestDiscoverer so that it ignores the tests and never fires the event - that way the exclusion logic comes under tests as well.